src/main/java/cc/polyfrost/oneconfig/internal/command/OneConfigCommand.java

```diff
@@ -32,104 +32,85 @@
 import cc.polyfrost.oneconfig.internal.gui.HudGui;
 import cc.polyfrost.oneconfig.libs.universal.ChatColor;
 import cc.polyfrost.oneconfig.libs.universal.UChat;
-import cc.polyfrost.oneconfig.utils.commands.annotations.*;
+import cc.polyfrost.oneconfig.utils.commands.annotations.Command;
+import cc.polyfrost.oneconfig.utils.commands.annotations.Descriptor;
+import cc.polyfrost.oneconfig.utils.commands.annotations.Greedy;
 import cc.polyfrost.oneconfig.utils.gui.GuiUtils;
 
 /**
  * The main OneConfig command.
  */
-@Command(value = "oneconfig", aliases = {"ocfg", "oneconfig"}, description = "Access the OneConfig GUI.")
+@Command(value = "oneconfig", aliases = {"ocfg"})
 public class OneConfigCommand {
 
-    @Main
-    private static void main() {
+    @Descriptor(value = "", description = "Opens the OneConfig GUI")
+    private void main() {
         GuiUtils.displayScreen(OneConfigGui.create());
     }
 
-    @SubCommand(value = "hud", description = "Open the OneConfig HUD config.")
-    private static class HUDSubCommand {
-        @Main
-        private static void main() {
-            GuiUtils.displayScreen(new HudGui());
-        }
+    @Descriptor(value = "", description = "Opens the OneConfig HUD configurator.", aliases = {"edithud"})
```

Comment:
I feel like descriptor should be something like @ Main here instead and value shouldn't be needed here

Reply:
they don't have to be annotated, its called @ Descriptor because it adds info to the help message. value is displayed in the help message as well if its present.